### PR TITLE
[SDWebImage] Updated to v4.4

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

**Comment by bholmes** [comment]
There appears to be a merge issue that was committed.  I doubt the library will build.  @adlair needs to get this PR building again.

File: iOS/SDWebImage/component/component.yaml [hunk 1]
```diff
@@ -1,31 +1,90 @@
...
-name: SDWebImage for Xamarin.iOS
+version: 4.0.0
+name: SDWebImage for Xamarin.iOS, Xamarin.tvOS, Xamarin.Mac
 id: sdwebimage
 publisher: Xamarin Inc
 publisher-url: https://xamarin.com/
 src-url: https://github.com/xamarin/XamarinComponents/tree/master/iOS/SDWebImage
+<<<<<<< HEAD
 summary: Extensions for UIImageView allowing you to asynchronously load web images.
+=======
```
Comment (bholmes) [inline iOS/SDWebImage/component/component.yaml]: This is a mistake that needs fixed

File: iOS/SDWebImage/samples/SDWebImageSimpleSample/SDWebImageSimpleSample/Info.plist [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
+		<string>armv7</string>
+	</array>
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+<<<<<<< HEAD
```
Comment (bholmes) [inline iOS/SDWebImage/samples/SDWebImageSimpleSample/SDWebImageSimpleSample/Info.plist]: This is a mistake that needs fixed

File: iOS/SDWebImage/component/component.yaml [hunk 3]
```diff
@@ -1,31 +1,90 @@
...
+      - Xamarin.SDWebImage
+      - Xamarin.SDWebImage.iOS
+      - Xamarin.SDWebImage.MacOS
+    installNuGets:
+      - project: SDWebImageSampleTV
+        packages:
+          - Xamarin.SDWebImage
+  - name: "iOS MapKit Annotation Image"
     path: "../samples/SDWebImageMapKitSample/SDWebImageMapKitSample.sln"
+<<<<<<< HEAD
```
Comment (bholmes) [inline iOS/SDWebImage/component/component.yaml]: This is a mistake that needs fixed

File: iOS/SDWebImage/nuget/SDWebImage.nuspec [hunk 4]
```diff
@@ -2,17 +2,24 @@
...
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>Extensions for UIImageView allowing you to asynchronously load web images.</description>
+    <description>Extensions for UIImageView and NSImageView allowing you to asynchronously load web images.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <projectUrl>https://go.microsoft.com/fwlink/?linkid=865081</projectUrl>
     <licenseUrl>https://go.microsoft.com/fwlink/?linkid=865072</licenseUrl>
   </metadata>
   <files>
+<<<<<<< HEAD
```
Comment (bholmes) [inline iOS/SDWebImage/nuget/SDWebImage.nuspec]: This is a mistake that needs fixed

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/Info.plist [hunk 5]
```diff
@@ -1,46 +1,52 @@
...
 	</array>
-	<key>UIMainStoryboardFile</key>
-	<string>Main</string>
-	<key>UIMainStoryboardFile~ipad</key>
-	<string>Main</string>
+	<key>XSAppIconAssets</key>
+	<string>Assets.xcassets/AppIcon.appiconset</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>We want to use your location, bwahahaha!</string>
+<<<<<<< HEAD
```
Comment (bholmes) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/Info.plist]: This is a mistake that needs fixed

File: iOS/SDWebImage/License.md~6b6330588e1d0c93dea410c83be94292d3f4402b [hunk 6]
```diff
@@ -0,0 +1,11 @@
+**The MIT License (MIT)**
```
Comment (bholmes) [inline iOS/SDWebImage/License.md~6b6330588e1d0c93dea410c83be94292d3f4402b]: This file is added by mistake

File: iOS/SDWebImage/License.md~HEAD [hunk 7]
```diff
@@ -0,0 +1,11 @@
+**The MIT License (MIT)**
```
Comment (bholmes) [inline iOS/SDWebImage/License.md~HEAD]: This file is added by mistake

**Review by bholmes — DISMISSED** [review]

**Comment by adlair** [comment]
Issues fixed.

**Review by newky2k — CHANGES_REQUESTED** [review]
when running  ->

sh ../../build.sh -t=samples

get this error



"/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample.sln" (Build target) (1) ->
"/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj" (default target) (2) ->
(_CoreCompileInterfaceDefinitions target) -> 
  /Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018: The "IBTool" task failed unexpectedly. [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018: System.ArgumentException: An item with the same key has already been added. Key: CFBundleName [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at System.ThrowHelper.ThrowAddingDuplicateWithKeyArgumentException (System.Object key) [0x00006] in <bb7b695b8c6246b3ac1646577aea7650>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) [0x000ad] in <bb7b695b8c6246b3ac1646577aea7650>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) [0x00000] in <bb7b695b8c6246b3ac1646577aea7650>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PDictionary.Add (System.String key, Xamarin.MacDev.PObject value) [0x00001] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PropertyListFormat+XmlFormat+Context.ReadDict (Xamarin.MacDev.PDictionary dict) [0x000dc] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PropertyListFormat+ReadWriteContext.ReadObject () [0x000ed] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PObject.FromFile (System.String fileName, System.Boolean& isBinary) [0x00042] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PDictionary.FromFile (System.String fileName, System.Boolean& isBinary) [0x00001] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.PDictionary.FromFile (System.String fileName) [0x00001] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Xamarin.MacDev.Tasks.IBToolTaskBase.Execute () [0x000b6] in <61a9397755bb4851982b9c1872ed8a1e>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute () [0x00023] in <61115f75067146fab35b10183e6ee379>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]
/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/iOS/Xamarin.iOS.Common.targets(1324,3): error MSB4018:   at Microsoft.Build.BackEnd.TaskBuilder+<ExecuteInstantiatedTask>d__26.MoveNext () [0x00212] in <61115f75067146fab35b10183e6ee379>:0  [/Users/davidhumphreys/Git/XamarinComponents/iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]

File: iOS/SDWebImage/nuget/SDWebImage.nuspec [hunk 8]
```diff
@@ -2,17 +2,21 @@
 <package>
   <metadata>
     <id>Xamarin.SDWebImage</id>
-    <title>SDWebImage for Xamarin.iOS</title>
-    <version>3.7.5</version>
+    <title>SDWebImage for Xamarin.iOS, Xamarin.tvOS, Xamarin.macOS</title>
+    <version>4.0.0-beta2</version>
```
Comment (newky2k) [inline iOS/SDWebImage/nuget/SDWebImage.nuspec]: should this be 4.4?

File: iOS/SDWebImage/component/component.yaml [hunk 9]
```diff
@@ -1,31 +1,84 @@
-version: 3.7.5
-name: SDWebImage for Xamarin.iOS
+version: 4.0.0
```
Comment (newky2k) [inline iOS/SDWebImage/component/component.yaml]: 4.4.0

**Review by newky2k — CHANGES_REQUESTED** [review]

File: iOS/SDWebImage/component/component.yaml (hunk 9 above)
Comment (adlair) [reply iOS/SDWebImage/component/component.yaml]: Fixed.

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 10]
```diff
@@ -18,14 +18,14 @@
     <DefineConstants>DEBUG;ENABLE_TEST_CLOUD;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Should be Automatic developer

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 11]
```diff
@@ -35,7 +35,7 @@
     <OutputPath>bin\iPhone\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Should Automatic developer

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 12]
```diff
@@ -49,7 +49,7 @@
     <OutputPath>bin\iPhoneSimulator\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 13]
```diff
@@ -64,7 +64,7 @@
     <DefineConstants>DEBUG;ENABLE_TEST_CLOUD;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleMac/SDWebImageSampleMac/SDWebImageSampleMac.csproj [hunk 14]
```diff
@@ -20,30 +20,36 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <EnableCodeSigning>false</EnableCodeSigning>
-    <CodeSigningKey>Mac Developer</CodeSigningKey>
+    <EnableCodeSigning>true</EnableCodeSigning>
+    <CodeSigningKey>Mac Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodeSigningKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleMac/SDWebImageSampleMac/SDWebImageSampleMac.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleMac/SDWebImageSampleMac/SDWebImageSampleMac.csproj [hunk 15]
```diff
@@ -20,30 +20,36 @@
...
+    <EnableCodeSigning>true</EnableCodeSigning>
     <CreatePackage>true</CreatePackage>
     <EnablePackageSigning>false</EnablePackageSigning>
     <IncludeMonoRuntime>true</IncludeMonoRuntime>
     <UseSGen>true</UseSGen>
     <UseRefCounting>true</UseRefCounting>
     <LinkMode>SdkOnly</LinkMode>
     <PlatformTarget>x64</PlatformTarget>
+    <AOTMode>None</AOTMode>
+    <CodeSigningKey>Mac Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodeSigningKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleMac/SDWebImageSampleMac/SDWebImageSampleMac.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj [hunk 16]
```diff
@@ -18,7 +18,7 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj [hunk 17]
```diff
@@ -35,7 +35,7 @@
     <OutputPath>bin\iPhone\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj [hunk 18]
```diff
@@ -51,7 +51,7 @@
     <OutputPath>bin\iPhoneSimulator\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj [hunk 19]
```diff
@@ -66,7 +66,7 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSampleTV/SDWebImageSampleTV/SDWebImageSampleTV.csproj]: Should an Automatic developer

File: iOS/SDWebImage/samples/SDWebImageSimpleSample/SDWebImageSimpleSample/SDWebImageSimpleSample.csproj [hunk 20]
```diff
@@ -18,7 +18,7 @@
     <DefineConstants>DEBUG;ENABLE_TEST_CLOUD;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (newky2k) [inline iOS/SDWebImage/samples/SDWebImageSimpleSample/SDWebImageSimpleSample/SDWebImageSimpleSample.csproj]: Should an Automatic developer

**Review by newky2k — CHANGES_REQUESTED** [review]

File: iOS/SDWebImage/nuget/SDWebImage.nuspec [hunk 21]
```diff
@@ -2,17 +2,21 @@
...
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>Extensions for UIImageView allowing you to asynchronously load web images.</description>
+    <description>Extensions for UIImageView and NSImageView allowing you to asynchronously load web images.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <projectUrl>https://go.microsoft.com/fwlink/?linkid=865081</projectUrl>
     <licenseUrl>https://go.microsoft.com/fwlink/?linkid=865072</licenseUrl>
   </metadata>
   <files>
     <file src="output/unified/SDWebImage.dll" target="lib/Xamarin.iOS" />
```
Comment (newky2k) [inline iOS/SDWebImage/nuget/SDWebImage.nuspec]: <file src="output/unified/SDWebImage.dll" target="lib/Xamarin.iOS" />

is line is no longer needed and doesn't work

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — CHANGES_REQUESTED** [review]
Need to make sure that the siging cert is set to automatic on all build configurations on all samples

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 22]
```diff
@@ -3,108 +3,101 @@
...
+    <Optimize>true</Optimize>
+    <OutputPath>bin\iPhone\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <ConsolePause>False</ConsolePause>
-    <CodesignKey>iPhone Developer</CodesignKey>
-    <MtouchArch>ARMv7, ARMv7s, ARM64</MtouchArch>
-    <MtouchI18n>
-    </MtouchI18n>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (Redth) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Needs fixing

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 23]
```diff
@@ -3,108 +3,101 @@
...
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\iPhoneSimulator\Release</OutputPath>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <ConsolePause>False</ConsolePause>
-    <CodesignKey>iPhone Distribution</CodesignKey>
-    <BuildIpa>True</BuildIpa>
-    <CodesignProvision>Automatic:AdHoc</CodesignProvision>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (Redth) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Needs fixing

File: iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj [hunk 24]
```diff
@@ -3,108 +3,101 @@
...
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\iPhone\Debug</OutputPath>
+    <DefineConstants>DEBUG;ENABLE_TEST_CLOUD;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <ConsolePause>False</ConsolePause>
-    <CodesignProvision>Automatic:AppStore</CodesignProvision>
-    <CodesignKey>iPhone Distribution</CodesignKey>
+    <CodesignKey>iPhone Developer: Jesus Adlair Cerecedo Mendez (83Q48VP5BY)</CodesignKey>
```
Comment (Redth) [inline iOS/SDWebImage/samples/SDWebImageMapKitSample/SDWebImageMapKitSample/SDWebImageMapKitSample.csproj]: Needs fixing, check everywhere for this please.

**Comment by adlair** [comment]
Bundle signing fixed.

**Review by newky2k — APPROVED** [review]

**Pull request merged**